pkg/transport/insecure.go

```diff
@@ -5,19 +5,23 @@ package transport
 
 import (
 	"context"
+	"crypto/tls"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
 
-// DialAddressInsecure returns an insecure grpc connection without tls to a node.
+// DialAddressEncrypted returns a grpc connection with encryption to a node; no client auth/cert is used.
 //
 // Use this method for communication with localhost. For example, with the inspector or debugging services.
 // Otherwise in most cases DialNode should be used for communicating with nodes since it is secure.
-func DialAddressInsecure(ctx context.Context, address string, opts ...grpc.DialOption) (conn *grpc.ClientConn, err error) {
+func DialAddressEncrypted(ctx context.Context, address string, opts ...grpc.DialOption) (conn *grpc.ClientConn, err error) {
```

Comment:
This needs some better name. `DialAddressClient`, `DialAddressInspector` or something like that... `Encrypted` probably isn't clear in what the intent of the func is.

Reply:
I get what you're saying but I'm having a hard time coming up with another name. `DialInspectorAddress` is currently the only thing it's being used for but it's not restricted to that usage. The most specific, non-usage related description is that it dials an encrypted connection.

Reply:
What about `DialAddressEncryptedInsecure`?

Reply:
DialAddress is already Encrypted. Maybe just DialAddressUnverified?

Reply:
Good point; I like that.